coq-builtin.elpi

```diff
@@ -910,6 +910,16 @@ external pred coq.hints.add-mode i:gref, i:string, i:list hint-mode.
 % GR
 external pred coq.hints.modes i:gref, i:string, o:list (list hint-mode).
 
+% [coq.hints.set-opaque C DB Opaque] Like Hint Opaque C : DB (or Hint
+% Transparent, if the boolean is ff).
+% Supported attributes:
+% - @local! (default: false)
+external pred coq.hints.set-opaque i:constant, i:string, i:bool.
+
+% [coq.hints.opaque C DB Opaque] Reads if constant C is opauqe (tt) or
```

Comment:
Just a typo:
```suggestion
% [coq.hints.opaque C DB Opaque] Reads if constant C is opaque (tt) or
```

Reply:
Thanks! (this file is generated, so I won't merge this code suggestion)